### PR TITLE
Adding a shortcut trigger for submitting an input with mentions

### PR DESCRIPTION
**Pull request by DavidHollins6**
### 💬 Description
The title says it all. We would like to submit the `InputWIthMentions` with cmd+enter.

### ✅ Checklist
- [ ] Tests written
- [ ] Browser tested
- [ ] Added to documentation
- [ ] Added to storybook

### CODE REVIEW

**Comment by DavidHollins6** [comment]
[CMD+ENTER should submit the workflow note form.](https://trello.com/c/5MgDp1gh/1719-cmdenter-should-submit-the-workflow-note-form)

File: lib/src/modules/input/InputWithMentions.js [hunk 1]
```diff
@@ -94,6 +98,28 @@ export function InputWithMentions({
         />
       </MentionsInput>
       <div ref={mentionsPortalRef} />
+      <ShortcutTrigger
```
Comment (kyleharper) [inline lib/src/modules/input/InputWithMentions.js:101]: 🔴   why do we need 2 `ShortcutTrigger`s? 

Can't we just set `withCtrlKey` & `withMetaKey` on a single component? 🤔

**Review by kyleharper — CHANGES_REQUESTED** [review]

File: lib/src/modules/input/InputWithMentions.js (hunk 1 above)
Comment (DavidHollins6) [reply lib/src/modules/input/InputWithMentions.js:101]: That would require the user to hold both `ctrl` and `meta` and `Enter` to trigger the shortcut.

Comment (kyleharper) [reply lib/src/modules/input/InputWithMentions.js:101]: haha oh yea! 😂

**Review by kyleharper — APPROVED** [review]

**Pull request merged**